src/test/java/com/ss/training/utopia/agent/ServiceTestSuite.java

```diff
@@ -47,4 +55,74 @@ public void readBookingsByAgentNoBookings() {
         assertEquals(bookingsByAgent.size(), 0);
     }
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+    @Test
+	public void cancelBooking() {
+        //Longs
+        Long oneLong = (long) 1;
+        Long twoLong = (long) 2;
+
+        // Times
+        final Long HOUR = (long) 3_600_000;
+        Long now = Instant.now().toEpochMilli();
+        Timestamp future = new Timestamp(now + HOUR);
+
+        // Future flight and booking - Booker ID: 1, only cancellable flight by user 1
+        Flight futureFlight = new Flight(oneLong, twoLong, future, oneLong, null, null);
+        Booking cancellableBooking = new Booking(oneLong, oneLong, oneLong, true, null);
+
+        flightDAO.save(futureFlight);
+        bookingDao.save(cancellableBooking);
+
+        List<Booking> foundBookings;
+        foundBookings = bookingDao.findCancellable(oneLong);
+        assertNotNull(foundBookings);
```

Comment:
I don't think this is necessary. If the list is null then the size assertion will catch it (same for line 112).